src/modules/dashboard/dashboard.hooks.ts

```diff
@@ -31,3 +45,31 @@ export const useGetRequestsList = (queryParams: IFetchComplaintListRequest) => {
     isLoading: isLoading || isFetching,
   };
 };
+
+export const usePostBulkUpdate = (
+  setpayloadload: (a: BulkUpload) => void,
```

Comment:
Instead of passing all state functions here pass a callback named onBulkUpdateSuccess which will be called here. & update the state inside the onBulkUpdateSuccess function